pom.xml

```diff
@@ -43,9 +43,19 @@
 			<version>1.47.1</version>
 		</dependency>
 		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+			<version>18.0</version>
+		</dependency>
+		<dependency>
 			<groupId>org.elasticsearch</groupId>
 			<artifactId>elasticsearch</artifactId>
-			<version>1.3.2</version>
+			<version>2.3.3</version>
+		</dependency>
+		<dependency>
+			<groupId>com.sun.jna</groupId>
+			<artifactId>jna</artifactId>
+			<version>3.0.9</version>
```

Comment:
Ah, the issue seems to be different versions of guava pulled in as transitive dependencies by com.google.gdata and elasticsearch (see "Dependency Hierarchy" tab in the Eclipse pom editor). Since we don't depend on guava directly, I think a better solution would be to exclude it in the com.google.gdata dependency.